docs/content.md

```diff
@@ -19,22 +27,12 @@ After installing Forge, simply put the downloaded `ReplayMod.jar` file in the `/
 > **Note:** The **Replay Mod** is entirely client side and can not be installed on the servers you play on.
 
 ## Installing FFmpeg [ffmpeg]
-To use the **Replay Mod**'s [Rendering Feature](#replaying-render), you need to have FFmpeg installed.
+To render your creations with **Replay Mod** you will need to have FFmpeg installed.
 
 ### Windows [windows]
-Download the latest **FFmpeg Static Build** for your architecture from <http://ffmpeg.zeranoe.com/builds/>.  
-Then, extract the downloaded `.7z` file in the folder you want to install FFmpeg in.
-
-Next, download this **Batch Script** to quickly install FFmpeg: <https://replaymod.com/files/ffmpeg-path-installer.bat>
+Download the latest FFmpeg build from <http://ffmpeg.zeranoe.com/builds/>. Make sure to select the latest release in the dropdown next to 'Version'. This will be a number like `4.3` or `4.3.1`. Chose the highest number.
```

Comment:
```suggestion
Download the latest FFmpeg build from <http://ffmpeg.zeranoe.com/builds/>. Make sure to select the latest release in the dropdown next to 'Version'. This will be a number like `4.3` or `4.3.1`. Choose the highest number.
```